twiliochat/MessagingManager.swift

```diff
@@ -1,177 +1,189 @@
+    }
+    
+    func presentViewControllerByName(viewController: String) {
+        presentViewController(controller: storyBoardWithName(name: "Main").instantiateViewController(withIdentifier: viewController))
+    }
+    
+    func presentLaunchScreen() {
+        presentViewController(controller: storyBoardWithName(name: "LaunchScreen").instantiateInitialViewController()!)
+    }
+    
+    func presentViewController(controller: UIViewController) {
+        let window = UIApplication.shared.delegate!.window!!
+        window.rootViewController = controller
+    }
+    
+    func storyBoardWithName(name:String) -> UIStoryboard {
+        return UIStoryboard(name:name, bundle: Bundle.main)
+    }
+    
+    // MARK: User and session management
+    
+    func loginWithUsername(username: String,
+                           completion: @escaping (Bool, NSError?) -> Void) {
+        SessionManager.loginWithUsername(username: username)
+        connectClientWithCompletion(completion: completion)
+    }
+    
+    func logout() {
+        SessionManager.logout()
+        DispatchQueue.global(qos: .userInitiated).async {
+            self.client?.shutdown()
+            self.client = nil
+        }
+        self.connected = false
+    }
+    
+    // MARK: Twilio Client
+    
+    func loadGeneralChatRoomWithCompletion(completion:@escaping (Bool, NSError?) -> Void) {
+        ChannelManager.sharedManager.joinGeneralChatRoomWithCompletion { succeeded in
+            if succeeded {
+                completion(succeeded, nil)
+            }
+            else {
+                let error = self.errorWithDescription(description: "Could not join General channel", code: 300)
+                completion(succeeded, error)
+            }
+        }
+    }
+    
+    func connectClientWithCompletion(completion: @escaping (Bool, NSError?) -> Void) {
+        if (client != nil) {
+            logout()
+        }
+        
+        requestTokenWithCompletion { succeeded, token in
+            if let token = token, succeeded {
+                self.initializeClientWithToken(token: token)
+                completion(succeeded, nil)
+            }
+            else {
+                let error = self.errorWithDescription(description: "Could not get access token", code:301)
+                completion(succeeded, error)
+            }
+        }
+    }
+    
+    func initializeClientWithToken(token: String) {
+        let accessManager = TwilioAccessManager(token:token, delegate:self)
+        
+        UIApplication.shared.isNetworkActivityIndicatorVisible = true
+        
+        TwilioChatClient.chatClient(withToken: token, properties: nil, delegate: self) { [weak self] result, chatClient in
+            
+            guard (result?.isSuccessful() ?? false) else { return }
+            
+            UIApplication.shared.isNetworkActivityIndicatorVisible = true
+            
+            self?.connected = true
+            self?.client = chatClient
+        }
```

Comment:
Hi @blender 

Thanks for opening this PR, I really appreciate your effort but, It's hard to see the differences with all those indentation changes, would you mind to separate your changes in two commits? one for aesthetics (like indentation, code-style) and another one with the real code changes?

Also, I think every reviewer would appreciate clear commits.
Thanks in advance.